inst/examples/03-formats.Rmd

```diff
@@ -218,9 +231,235 @@ favicon: "favicon.ico"
 
 A nice effect of setting `description` and `cover-image` is that when you share the link of your book on some social network websites such as Twitter, the link can be automatically expanded to a card with the cover image and description of the book.
 
-### Bootstrap style
+### Three-column Bootstrap style {#bs4-book}
+
+The `bs4_book()` output format is built with Bootstrap (<https://getbootstrap.com>), using carefully crafted features to provide a clean reading experience whether you are on a phone, tablet, or desktop. On a full-size screen, the layout includes three columns of content so readers can quickly see the book chapters as links on the left-hand side, and within-chapter content links in the right-hand sidebar. You can read an example book here: <https://mastering-shiny.org/>
+
+```{r fig.align='center', echo=FALSE, include=identical(knitr:::pandoc_to(), 'html'), fig.link='https://mastering-shiny.org/', fig.alt="Home page for a bs4_book showing the layout with a table of contents on the left, main chapter content in the center, and an 'on this page' sidebar on the right."}
+knitr::include_graphics('images/bs4_book.png', dpi = NA)
+```
+
+This output format builds upon the Bootstrap CSS framework (version 4: <https://getbootstrap.com/docs/4.0/>), an open source library of reusable chunks of HTML, CSS, and JavaScript code.
```

Comment:
Duplicates the first sentence of the first paragraph a bit. Could they be combined?